vendors/circom/circomlib/build/compile_witness_generator.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+function main() {
+  CIRCOMLIB=bazel-bin/circomlib/build/compile_witness_generator.runfiles/kroma_network_circom/circomlib
+  cd ${CIRCOMLIB}/build
+  cp ../../../iden3_circom/code_producers/src/c_elements/common/* .
+  if [ ! -z "$2" ]; then
+    echo "Using $2"
+    cp ../../../iden3_circom/code_producers/src/c_elements/$2/* .
+  else
+    echo "No field specified! Using bn128 as default. Supported: bn128, bls12381"
```

Comment:
Please sort this in alphabetical order.

```suggestion
    echo "No field specified! Using bn128 as default. Supported: bls12381, bn128"
```